src/evaluator/rollout/envs.py

```diff
@@ -153,15 +152,13 @@ class MultiViewImageObsWrapper(ObservationWrapper):
     def __init__(
         self,
         env: gym.Env,
-        image_size: tuple[int, int] = (64, 64),
         camera_attribute: str | None = "camera_name",
-        camera_names: tuple[str, ...] = ("topview", "corner3", "gripperPOV"),
+        camera_names: tuple[str, ...] = ("corner",),
```

Comment:
nitpick: remove leading comma inside the parentheses
```suggestion
    camera_names: tuple[str, ...] = ("corner")
```